deps/rabbitmq_auth_backend_oauth2/src/rabbit_auth_backend_oauth2.erl

```diff
@@ -82,7 +73,8 @@ check_vhost_access(#auth_user{impl = DecodedTokenFun},
                    VHost, _AuthzData) ->
     with_decoded_token(DecodedTokenFun(),
         fun(_Token) ->
-            Scopes      = get_scopes(DecodedTokenFun()),
+            DecodedToken = DecodedTokenFun(),
```

Comment:
perhaps we should just not line equal signs up and just have a single space before the `=` everywhere?